creator-node/src/RehydrateIpfsQueue.js

```diff
@@ -27,29 +75,35 @@ class RehydrateIpfsQueue {
       }
     )
 
+    this.RehydrateRedisCounter = new RehydrateRedisCounter(redisClient)
+
     // Most errors in the rehydrate calls will be caught; this try/catch is to catch unexpected errors
 
     this.queue.process(PROCESS_NAMES.rehydrate_file, config.get('rehydrateMaxConcurrency'), async (job, done) => {
+      const taskType = `rehydrateIpfsFromFsIfNecessary`
```

Comment:
lets move this taskType into the global scope so it doesn't have to keep getting recreated when this callback is made

Reply:
good idea